edb/server/main.py

```diff
@@ -217,6 +223,51 @@ async def _run_server(
             await sc.wait_for(ss.stop())
 
 
+def _init_tls_certs(
+    args: srvargs.ServerConfig,
+    cert_dir: Optional[tempfile.TemporaryDirectory],
+):
+    cert_path = None
+    if args.tls_cert_file:
+        cert_file = pathlib.Path(args.tls_cert_file).resolve()
+        if args.tls_key_file is None:
+            return str(cert_file), None
```

Comment:
I'm just curious -- why not just return Path objects?

Reply:
Oh yes, `Path` objects would do better! It's been a few versions of changes and I cannot remember how it ended up in strings. Let me push a fix.